include/zrepl_mgmt.h

```diff
@@ -74,6 +74,7 @@ typedef struct inject_delay_s {
 	int pre_uzfs_write_data;
 	int downgraded_replica_rebuild_size_set;
 	int io_receiver_exit;
+	uint64_t dw_replica_rebuild_error_io;
```

Comment:
this structure is actually related to injecting delays.. probably, for this, a different data structure would be better

Reply:
ok